src/tequila/simulators/simulator_cirq.py

```diff
@@ -9,10 +9,27 @@
 
 import cirq
 
+cirq_devices = {'foxtail':cirq.google.Foxtail,
+                'sycamore':cirq.google.Sycamore,
+                'sycamore23':cirq.google.Sycamore23,
+                'bristlecone':cirq.google.Bristlecone
+               }
 
 map_1 = lambda x: {'exponent':x}
 map_2 = lambda x: {'exponent':x/np.pi,'global_shift':-0.5}
 
+def check_cirq_device(device):
```

Comment:
make `check_device` member function (see simulator_api comment) same for pyquil and qiskit